.gitignore

```diff
@@ -305,4 +305,9 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
+# Additional
+*.json
+*.yaml
```

Comment:
Do we really need to exclude a config with only:

    LevelDBLocation: "./db"

in it?

Reply:
And anyway broad ignoring of all `.json` and `.yaml` files is not an option

Reply:
Config file will grow in the future so It should be in gitignore for future needs, just to make sure that onone in the future will forget to add it. 

I changed this global ignore to only files that contain config in it, in works for now but feel free to change it if needed or in case you found a better solution